app/javascript/admin/controllers/confirmation_modal_controller.js

```diff
@@ -0,0 +1,80 @@
+import { Controller } from 'stimulus';
+import { adminModal } from '../adminModal';
+
+const confirmationText = (username) =>
+  `My username is @${username} and this action is 100% safe and appropriate.`;
+
+export default class ConfirmationModalController extends Controller {
+  static targets = [
+    'confirmationModalAnchor',
+    'confirmationTextField',
+    'confirmationTextWarning',
+  ];
+
+  closeConfirmationModal() {
+    this.confirmationModalAnchorTarget.innerHTML = '';
+    document.body.style.height = 'inherit';
+    document.body.style.overflowY = 'inherit';
+  }
+
+  async sendToEndpoint({ itemId, endpoint }) {
```

Comment:
I'm facing frustration issues with my `sendToEndpoint` function 😓 

Here is the goal: hit the endpoint specified, in order to delete the chosen item, then have the page refresh and show the appropriate flash notice (all this is in the RFC).

My issue is: when I click `Confirm changes` in the modal, nothing seems to happen (the modal remains visible) and I get the following failure in the console:
```
DELETE http://localhost:3000/admin/content_manager/badge_achievements 404 (Not Found)
```

However, when I manually refresh the page, the achievement I deleted is gone and the expected flash notice appears at the page's top.

**I truly do not understand why this is so. Please help if you can.** 🙏🏾 

Here is a video depicting what I described above:  https://www.loom.com/share/6a418555d13f4180a70317b5cfd248d5